automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterFixtureSpec.scala

```diff
@@ -13,7 +13,7 @@ import org.broadinstitute.dsde.workbench.google2.MachineTypeName
 /**
  * trait BeforeAndAfterAll - One cluster per Scalatest Spec.
  */
-abstract class ClusterFixtureSpec extends fixture.FreeSpec with BeforeAndAfterAll with LeonardoTestUtils with Retries {
+abstract class ClusterFixtureSpec  extends fixture.FreeSpec with BeforeAndAfterAll with LeonardoTestUtils with Retries with GPAllocBeforeAndAfterAll{
```

Comment:
`GPAllocBeforeAndAfterAll` here is not needed

Reply:
Fixed